Gemfile

```diff
@@ -12,6 +12,7 @@ gem "awesome_print"
 gem "bootsnap", require: false
 gem "config"
 gem "cssbundling-rails"
+gem "csv"
```

Comment:
Does this mean we can remove `require "csv"` across the app?

Reply:
Think so!